certif/app/components/new-certification-candidate-modal.js

```diff
@@ -66,8 +66,10 @@ export default class NewCertificationCandidateModal extends Component {
 
   @action
   updateComplementaryCertification(complementaryCertification) {
-    if (complementaryCertification?.target?.value !== 'none') {
+    if (complementaryCertification?.key) {
```

Comment:
Du coup ligne 326 du fichier `hbs` je propose de supprimer le `value="none"` qui n'a plus d'utilité.
Dans l'idée ou soit on passe 'by value' ou par 'on input'. Ici on est parti sur du 'on input'.

Si on passait 'by value' alors il faut ici faire (si value prenait par exemple l'id)

```suggestion
   const selectedComplementary = this.currentUser.currentAllowedCertificationCenterAccess?.habilitations.find((habilitation) => habilitation.id === complementaryCertification?.target?.value))
    if (selectedComplementary) {
    ... this.args... = selectedComplementary;
```

La version 'on input' ici marche très bien ceci dit :)